library/include/G4CMPCrystalGroup.hh

```diff
@@ -13,6 +13,7 @@
 // 20170525  M. Kelsey -- Add default "rule of five" copy/move operators
 // 20170728  Change function args "alpha, beta, gamma" to "al, bt, gm" (-Wshadow)
 // 20240416  S. Zatschler -- Change function arg "g" to "c" (-Wshadow)
+// 20240924  I. Hernandez -- add    G4bool FillHexagonal(G4double Cij[6][6]) const;
```

Comment:
Maybe make this a bit more narrative?  "Add functionality to support hexagonal crystal structures."